Libplanet.Explorer/GraphTypes/BlocksQuery.cs

```diff
@@ -59,6 +60,20 @@ public BlocksQuery(BlockChain<T> chain)
                 }
             );
 
+            Field<TransactionType<T>>(
+                "transaction",
+                arguments: new QueryArguments(
+                    new QueryArgument<IdGraphType> { Name = "id" }
+                ),
+                resolve: context =>
+                {
+                    TxId id = new TxId(
```

Comment:
We can omit type here because `id`'s type is obvious in this context.

```suggestion
                    var id = new TxId(
```